scene/gui/video_player.cpp

```diff
@@ -102,6 +102,10 @@ void VideoPlayer::_mix_audio() {
 			}
 
 		} break;
+		case AudioServer::SPEAKER_SURROUND_31: {
+
+			// FIXME: Implement
+		} break;
```

Comment:
```
scene/gui/video_player.cpp:94:9: warning: enumeration value 'SPEAKER_SURROUND_31' not handled in switch [-Wswitch]
```
@reduz @marcelofg55 Doesn't seem to be implemented anywhere, should it be simply dropped?